spec/requests/compliments_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-describe 'Compliments API' do
+RSpec.describe 'Compliments API', type: :request do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.